CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * [FEATURE] Ingester: Add active series to all_user_stats page. #4972
 * [FEATURE] Ingester: Added `-blocks-storage.tsdb.head-chunks-write-queue-size` allowing to configure the size of the in-memory queue used before flushing chunks to the disk . #5000
 * [FEATURE] Query Frontend: Log query params in query frontend even if error happens. #5005
+* [FEATURE] Experimental Ingester: Enable snapshotting of In-memory TSDB on disk during shutdown via `-blocks-storage.tsdb.memory-snapshot-on-shutdown`. And we can also control the behavior of a per user through `blocks-storage.tsdb.memory-snapshot-on-shutdown-per-user`. #5011
```

Comment:
The intent is to make this flag experimental?

If so i think we should hide it from the documentation as we did with the experimental other flags (like vertical query sharding) and document on : https://cortexmetrics.io/docs/configuration/v1guarantees/

On the other hand I think its ok for this flag no be experimental (this feature is on prometheus for some time now).

Reply:
It is still experimental in Prometheus. Let's keep it experimental for now.

Reply:
So i think we should hide from the documentation and add in the [v1guarantees/](https://cortexmetrics.io/docs/configuration/v1guarantees/) doc.

Reply:
> So i think we should hide from the documentation and add in the [v1guarantees/](https://cortexmetrics.io/docs/configuration/v1guarantees/) doc.

Ok. I'm new here and haven't found a better place before. :)